### DISCUSSION TITLE
Question on Route Based VPN Documentation

### DISCUSSION THREAD

**Discussion by tet-web-dev** [strongswan/strongswan#1922] [Q&A] [answered]
Why does the [example](https://docs.strongswan.org/docs/5.9/features/routeBasedVpn.html#_example_creation_of_two_vti_devices_vti0_and_ipsec0) provide two tunnels? one vti and another ipsec0?

Are two tunnels required for the VPN to work properly or are two created for example of a setup that would need two tunnels? 

My best guess is one tunnel is for the initial connection between the peers and the other is encrypted traffic? or maybe one is return traffic? Really confused on this part. Thanks!

**Comment by tobiasbrunner** [comment, marked as answer]
> Why does the [example](https://docs.strongswan.org/docs/5.9/features/routeBasedVpn.html#_example_creation_of_two_vti_devices_vti0_and_ipsec0) provide two tunnels? one vti and another ipsec0?

Both are VTI devices, just different names. It's just an example on how to create and configure them.

> Are two tunnels required for the VPN to work properly or are two created for example of a setup that would need two tunnels?

No, why would you think that? How many are necessary really only depends on the use case.


> My best guess is one tunnel is for the initial connection between the peers and the other is encrypted traffic?

No, IKE traffic is not going via VTI device. You might want to read the [introduction](https://docs.strongswan.org/docs/5.9/howtos/introduction.html) and the complete route-based doc.

**Reply by tet-web-dev** [reply]
Got it, so the VTI/XFRM/GRE tunnels terminate the encrypted tunnel. 

I never would have guessed a "Virtual _Tunnel_ Interface" in the context of VPNs would not have the responsibility of moving encrypted traffic. 

Thanks for clearing those items up.
